sdk/digitaltwins/Azure.DigitalTwins.Core/tests/AdtQueryBuilderTests/AdtQueryBuilderTests.cs

```diff
@@ -92,5 +92,86 @@ public void AdtQueryBuilder_Select_Override()
                 .Should()
                 .Be("SELECT TOP(3) Room, Temperature FROM DigitalTwins");
         }
+
+        [Test]
+        public void AdtQueryBuilder_Where_Comparison()
+        {
+            new AdtQueryBuilder()
+                .Select("*")
+                .From(AdtCollection.DigitalTwins)
+                .WhereComparison("Temperature", QueryComparisonOperator.GreaterOrEqual, "50")
+                .Build()
+                .Stringify()
+                .ToUpper()
+                .Should()
+                .Be("SELECT * FROM DIGITALTWINS WHERE TEMPERATURE >= 50");
+        }
+
+        [Test]
+        public void AdtQueryBuilder_Where_Contains()
+        {
+            new AdtQueryBuilder()
+                .Select("*")
+                .From(AdtCollection.DigitalTwins)
+                .WhereContains("Location", QueryContainOperator.NIN, new string[] { "Paris", "Tokyo", "Madrid", "Prague" })
+                .Build()
+                .Stringify()
+                .ToUpper()
+                .Should()
+                .Be("SELECT * FROM DIGITALTWINS WHERE LOCATION NIN ['PARIS', 'TOKYO', 'MADRID', 'PRAGUE']");
```

Comment:
Uh, why is the entire query uppercased, even user-data?

Reply:
It was meant to just sanitize the string to make sure there weren't case errors since (to my knowledge) the ADT query lang isn't case sensitive. However, I see the point about user data and additionally don't think it's worth to do anymore, so I'll revert the instances of that.